flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/StatementSet.java

```diff
@@ -51,7 +51,7 @@
 	 * all statements and Tables.
 	 *
 	 * @param extraDetails The extra explain details which the explain result should include,
-	 *                     e.g. estimated cost, changelog mode for streaming
+	 *                     e.g. estimated cost, changelog mode for streaming, execution plan in json format
```

Comment:
```suggestion
	 *                     e.g. estimated cost, changelog mode for streaming, displaying execution plan in json format
```

Reply:
We should also update the docs of `TableEnviroment#explainSql()` and `Planner#explain`

Reply:
@godfreyhe Thanks for your reminder. I will update it.